### PR TITLE
Ansible module to create and delete Couchbase buckets

### PR DESCRIPTION
**Pull request by renaulth**
Ansible module to create and delete Couchbase buckets, using the Couchbase REST API.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks for submitting your module to Ansible Extras. Our policy is that each new module needs to be reviewed and approved by at least two official module reviewers, the list of whom can be found here: https://github.com/ansible/ansible-modules-extras/blob/devel/REVIEWERS.md 

If you know someone on that list, please poke them and encourage them to review your module. :)

To ensure that your module has the best chance of being approved, please double-check that you adhere to the Ansible module guidelines: http://docs.ansible.com/developing_modules.html#module-checklist

**Comment by robynbergeron** [comment]
Adding new process. We will be evaluating all new module PRs according to this process, effective immediately. 

# 

Thanks for submitting this new module to Ansible Extras! This module is now in community review, a process that is open to all Ansible users. In order for this module to be approved, it must gain the following votes:

“works_for_me”: If you have tested the module thoroughly, including testing of all of the module’s options, and if the module works for you, please add “works_for_me” in the comments.

“passes_guidelines”: If you have gone through the module guidelines and the module meets all of the requirements, please add “passes_guidelines” in the comments. Guidelines are available here: http://docs.ansible.com/developing_modules.html#module-checklist

“needs_revision”: If the module fails to work for you, or if it doesn’t meet guidelines, please add “needs_revision” in the comments with details about what needs to be fixed.

When a module has both “works_for_me” and “passes_guidelines” tags, we will promote the module for inclusion in Ansible Extras.  At this point, you will be expected to maintain the module by fixing bugs and evaluating pull requests in a timely manner. 

Thanks again for submitting your Ansible module!

File: database/couchbase_bucket.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+description:
+    - Creates or removes buckets on the Couchbase server using the Couchbase
+            REST API.
+        'state' = 'present' and 'force' = 'no', the bucket will be created
+            if it does not exist.
+        'state' = 'present' and 'force' = 'yes', the bucket will be deleted
+            and recreated if it does exist.
+        'state' = 'absent' and 'force' = 'no', the bucket will be deleted.
+        'state' = 'absent' and 'force' = 'yes', the bucket will be deleted.
+version _added: "1.7"
```
Comment (gregdek) [inline database/couchbase_bucket.py:32]: This is clearly no longer right, but we can always change it if this gets closer

**Comment by gundalow** [comment]
I've poked some people at CouchBase to have a look at this

File: database/couchbase_bucket.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+author: Renault Hess
```
Comment (gregdek) [inline database/couchbase_bucket.py:33]: Please add your Github ID to this

**Pull request closed without merging**